go/oasis-net-runner/fixtures/default.go

```diff
@@ -115,9 +115,9 @@ func newDefaultFixture() (*oasis.NetworkFixture, error) {
 	usingKeymanager := len(viper.GetString(cfgKeymanagerBinary)) > 0
 
 	if viper.GetBool(cfgSetupRuntimes) {
-		fixture.Runtimes = []oasis.RuntimeFixture{
+		if usingKeymanager {
 			// Key manager runtime.
-			{
+			keymanagerFixture := oasis.RuntimeFixture{
```

Comment:
Since you don't use `keymanagerFixture` anywhere else, you could simplify by doing:
```suggestion
			fixture.Runtimes = append(fixture.Runtimes, oasis.RuntimeFixture{
```